<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QWidget" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>10</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Deposit amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dep_amount">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>10</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="int_rate">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>100</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Term of placement</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="place_term">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>55</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="month_year">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>55</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Years</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>145</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Tax rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>145</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="pay_freq_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Payments freq</string>
   </property>
  </widget>
  <widget class="QComboBox" name="paym_freq">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>190</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentText">
    <string>Every day</string>
   </property>
   <item>
    <property name="text">
     <string>Every day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every week</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Once a month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every 6 month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>At the end</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="capitalisation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Interests capitalization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Replen. amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Replen. freq</string>
   </property>
  </widget>
  <widget class="QComboBox" name="repl_freq">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>265</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentText">
    <string>None</string>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Once a month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every 6 month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every year</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="rep_amo">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>310</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>310</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>245</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="withdr_freq">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>355</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentText">
    <string>None</string>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Once a month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every 6 month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every year</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>355</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Withdrawal freq</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Withdr. amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>400</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="withd_amo">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>400</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="llaabb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Accrued interest</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="acc_int">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>470</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>470</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>515</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Tax amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>515</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>515</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>405</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Total amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>560</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>560</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>610</y>
     <width>227</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: #acd4b4;
  color: white; 
  border-radius: 20px;
}

QPushButton:pressed {
    background-color: #8fba8f;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>610</y>
     <width>227</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: #acd4b4;
  color: white; 
  border-radius: 20px;
}

QPushButton:pressed {
    background-color: #8fba8f;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>dep_amount</zorder>
  <zorder>int_rate</zorder>
  <zorder>label_4</zorder>
  <zorder>place_term</zorder>
  <zorder>month_year</zorder>
  <zorder>label_3</zorder>
  <zorder>tax_rate</zorder>
  <zorder>label_7</zorder>
  <zorder>pay_freq_label</zorder>
  <zorder>paym_freq</zorder>
  <zorder>capitalisation</zorder>
  <zorder>label_9</zorder>
  <zorder>label_11</zorder>
  <zorder>repl_freq</zorder>
  <zorder>rep_amo</zorder>
  <zorder>label_10</zorder>
  <zorder>line</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>withd_amo</zorder>
  <zorder>withdr_freq</zorder>
  <zorder>line_2</zorder>
  <zorder>llaabb</zorder>
  <zorder>acc_int</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>tax_amount</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>total</zorder>
  <zorder>label_20</zorder>
  <zorder>clear</zorder>
  <zorder>calculate</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
